openstudiocore/src/model/Model_Impl.hpp

```diff
@@ -165,6 +169,11 @@ namespace detail {
         const std::shared_ptr<openstudio::detail::WorkspaceObject_Impl>& originalObjectImplPtr,
         bool keepHandle) override;
 
+    /// Set the WorkflowJSON
+    bool setWorkflowJSON(const WorkflowJSON& workflowJSON);
+
+    void resetWorkflowJSON();
+
```

Comment:
Well yes, I think it is useful to be able to use a WorkflowJSON without a Model.  But you can't have a Model without a WorkflowJSON.  I am not really sure what happens when you set a workflow json on a model, probably shouldn't be able to?  Saving model and saving workflow json are also related.